tests/api.rs

```diff
@@ -273,6 +276,86 @@ mod tests {
         assert_eq!(res.errors, None);
     }
 
+    #[test]
+    /// Tests links between components
+    fn api_graphql_component_links() {
```

Comment:
This test already existed. I've just renamed it to fit the layout of the other tests.